contract_function_result.go

```diff
@@ -108,14 +108,6 @@ func _ContractFunctionResultFromProtobuf(pb *services.ContractFunctionResult) Co
 		infos[i] = _ContractLogInfoFromProtobuf(info)
 	}
 
-	createdContractIDs := make([]ContractID, 0)
```

Comment:
Not sure why this is being removed.

Reply:
Because is was deprecated in protobuf, and lint was complaining that I am still using deprecated variable.